### DISCUSSION TITLE
Is it possible to create an issue that talks about open_spiel and webAssembly?

### DISCUSSION THREAD

**Discussion by LucasCelestinoSE** [deepmind/open_spiel#1176] [Ideas]
Are there any rules about discussing tools to complement open_spiel?
I noticed that there isn't much material on how to use open_spiel and webassembly together to create scalable games for the web, I wanted to know if it would be cool to create a discussion or issue about this.

**Comment by lanctot** [comment]
> Are there any rules about discussing tools to complement open_spiel? 

Nope, feel free to.  But better to do that here (in the discussion section) than via the issues, which should be used for OpenSpiel issue tracking.

> I noticed that there isn't much material on how to use open_spiel and webassembly together to create scalable games for the web, I wanted to know if it would be cool to create a discussion or issue about this.

Huh, TIL about webassembly. Interesting. Seems like there is a subreddit for it: https://www.reddit.com/r/WebAssembly/  . Maybe try mentioning it there too?
